<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*通用样式表*/
QPushButton{
	border:noe;
}

.QWidget{		/*只应用于当前QWidget对象*/
	background-color: rgb(255, 255, 255);
}

QComboBox,QLineEdit{
	font:normal 18px &quot;楷体&quot;;
	border:none;
	border-bottom:1px solid rgb(204, 204, 204);
}

QComboBox:hover,QLineEdit:hover{
	border-bottom:1px solid rgb(103, 103, 103);
}

QComboBox:focus,QLineEdit:focus{
	border-bottom:1px solid rgb(0, 170, 255);
}

/*指定样式表*/
QPushButton#reduce_Btn{
	image:url(:/new/icons/icons/becurrent/minus.png);
}

QPushButton#clos_Btn{
	image: url(:/new/icons/icons/becurrent/close.png);
}

QPushButton#reduce_Btn:hover{
	background-color: rgb(217, 217, 217);
}

QPushButton#clos_Btn:hover{
	background-color: rgb(255, 0, 0);
}

QPushButton#logon_Btn{
	border-radius:5px;
	background-color: rgb(0, 170, 255);
	font:normal 18px &quot;楷体&quot;;
	color:rgb(255, 255, 255);
}

QPushButton#logon_Btn:hover{
	background-color: rgb(0, 85, 255);
}

QPushButton#register_Btn{
	color:rgb(129, 129, 129);
}

QPushButton#register_Btn:hover{
	color: rgb(0, 0, 0);
}

QComboBox#usernumber_Box::drop-down{
	min-width: 10px;
	border-image: url(:/new/icons/icons/becurrent/arrow-down.png);
}

QComboBox#usernumber_Box::drop-down:hover{
    min-width: 10px;
	border-image: url(:/new/icons/icons/becurrent/arrow-down-blue.png);
}

QLabel#Top{
	background-color: rgb(0, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="Top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>130</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>130</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QPushButton" name="register_Btn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>175</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logon_Btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>240</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>安全登录</string>
    </property>
    <property name="icon">
     <iconset resource="../image/Image.qrc">
      <normaloff>:/new/icons/icons/becurrent/security.png</normaloff>:/new/icons/icons/becurrent/security.png</iconset>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QComboBox" name="usernumber_Box">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>340</width>
      <height>30</height>
     </size>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_Edit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>282</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>11</number>
     </property>
     <item>
      <widget class="QCheckBox" name="saved_number">
       <property name="text">
        <string>记住账号</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="saved_Password">
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="register_Btn_2">
       <property name="text">
        <string>找回密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="clos_Btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="reduce_Btn">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="user_image">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../image/Image.qrc">:/new/user/user/My.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
